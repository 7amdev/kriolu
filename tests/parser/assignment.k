a = 1 + 2 * 3 - 4;
y = 1 + 2 + 3 + 4 * 5 + 6 - 7;
a = b + 3 * 4 - 1;
