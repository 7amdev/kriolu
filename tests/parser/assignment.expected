<expression statement>
  <Assignment>
    a
    (1.0 + ((2.0 * 3.0) - 4.0))
<expression statement>
  <Assignment>
    y
    (1.0 + (2.0 + (3.0 + ((4.0 * 5.0) + (6.0 - 7.0)))))
<expression statement>
  <Assignment>
    a
    ((get b) + ((3.0 * 4.0) - 1.0))